src/main/java/com/agnieszkapawska/flashcards/domain/facades/FlashcardFacade.java

```diff
@@ -11,20 +12,20 @@
 import lombok.AllArgsConstructor;
 import org.modelmapper.ModelMapper;
 import org.springframework.dao.DataIntegrityViolationException;
-import java.util.HashSet;
-import java.util.Set;
+import java.util.*;
+import java.util.stream.Collectors;
```

Comment:
Do you need this line if you have the one above?

Reply:
Probably...